src/setup.rs

```diff
@@ -1,15 +1,19 @@
+use axum::extract::State;
+use sqlx::{Connection, Pool, Postgres};
```

Comment:
```suggestion
use sqlx::{Pool, Postgres};
```
`axum::extract::State` & `sqlx::Connection` are unused.